### PR TITLE
remove unnecessary fields

### PR DESCRIPTION
**Pull request by deedee**

### CODE REVIEW

File: src/main/java/com/appirio/service/challengefeeder/manager/ChallengeFeederManager.java [hunk 1]
```diff
@@ -232,32 +232,35 @@ public void pushChallengeFeeder(ChallengeFeederParam param) throws SupplyExcepti
...
                         .findFirst().orElse(null);
                 challenge.setContestId(Long.valueOf(cm.get("contestId").toString()));
                 challenge.setComponentId(Long.valueOf(cm.get("componentId").toString()));
                 //remove submitter if there's we'll populate from legacy mm data below
-                challenge.setResources(challenge.getResources().stream()
-                        .filter(r -> !"Submitter".equals(r.getRole())).collect(Collectors.toList()));
+// exclude this for now
+//                challenge.setResources(challenge.getResources().stream()
+//                        .filter(r -> !"Submitter".equals(r.getRole())).collect(Collectors.toList()));
                 challenge.setNumSubmissions(0L);
```
Comment (skyhit) [inline src/main/java/com/appirio/service/challengefeeder/manager/ChallengeFeederManager.java]: @deedee this is just related to mm?

Comment (deedee) [reply src/main/java/com/appirio/service/challengefeeder/manager/ChallengeFeederManager.java]: On that block yes. But resources field for all (see above) has been comment out

Comment (skyhit) [reply src/main/java/com/appirio/service/challengefeeder/manager/ChallengeFeederManager.java]: @deedee can we get the numbers correct?

Comment (skyhit) [reply src/main/java/com/appirio/service/challengefeeder/manager/ChallengeFeederManager.java]: @deedee ha, I see

**Pull request merged**